ims/einstein/operations.py

```diff
@@ -430,20 +430,42 @@ def list_all_images(self):
 
     @log
     def import_ceph_image(self, img):
+        """ Import an image in ceph to be used by BMI
```

Comment:
There is a second import in same file so add some more specificity to the docstring and also probably document the second one also in this PR.